src/dspbase.jl

```diff
@@ -432,15 +437,38 @@ function unsafe_conv_kern_os_edge!(
                     out_start .+ data_offset .+ save_blocksize .- 1,
                     out_stop
                 )
+                # print("data offset", data_offset, "\n")
```

Comment:
Looks like you don't know about [`@show`](https://docs.julialang.org/en/v1/base/base/#Base.@show), it's great!